core/src/main/scala/Interval.scala

```diff
@@ -0,0 +1,36 @@
+package helm
+
+import scalaz._, Scalaz._
+import argonaut._, Argonaut._
+
+sealed abstract class Interval
+
+/** Some types to represent check intervals, et al. */
+object Interval {
+
+  // So I considered using Scala's FiniteDuration type, but it
+  // seems like overkill.
```

Comment:
My first instinct was to argue against this, but I guess it gives you a nice way to centralize the serialization into Go durations as more pop up in the future.